beacon_node/beacon_chain/src/data_availability_checker/overflow_lru_cache.rs

```diff
@@ -479,9 +507,17 @@ impl<T: BeaconChainTypes> DataAvailabilityCheckerInner<T> {
         // Merge in the blobs.
         pending_components.merge_blobs(fixed_blobs);
 
+        if data_column_recv.is_some() {
+            // If `data_column_recv` is `Some`, it means we have all the blobs from engine, and have
+            // started computing data columns. We store the receiver in `PendingComponents` for
+            // later use when importing the block.
+            pending_components.data_column_recv = data_column_recv;
```

Comment:
If `data_column_recv` is already Some means we have double reconstruction happening. Should we mind that case or do something?

Reply:
Yes, I chatted with Michael about this - so currently it's possible any of these three happens at the same time
- gossip block triggering EL `getBlobs` and column computation
- rpc block triggering EL `getBlobs` and column computation
- gossip/rpc column triggering column reconstruction (only one computation at a time)

I initially wanted to address these altogether, but it got a bit messy and I decided to leave this issue to a separate PR and to keep this PR small , as I wanted to get this PR merged first and start interop testing.

Reply:
Created issue https://github.com/sigp/lighthouse/issues/6764